src/omv/modules/py_ml.c

```diff
@@ -40,20 +40,53 @@ static size_t py_ml_tuple_sum(mp_obj_tuple_t *o) {
 static void py_ml_process_input(py_ml_model_obj_t *model, mp_obj_t arg) {
     mp_obj_list_t *input_list = MP_OBJ_TO_PTR(arg);
 
-    for (size_t i = 0; i < model->inputs_size; i++) {
```

Comment:
Please revert inputs_size and outputs_size, it's an additional 4 bytes per Model object.

Reply:
Done